modules/virt-hot-unplugging-bridge-network-interface-cli.adoc

```diff
@@ -18,6 +18,7 @@ Hot unplugging is not supported for Single Root I/O Virtualization (SR-IOV) inte
 * Your VM must be running.
 * The VM must be created on a cluster running {VirtProductName} 4.14 or later.
 * The VM must have a bridge network interface attached.
+* You have permission to create and list `VirtualMachineInstanceMigration` objects.
```

Comment:
Maybe it's worth adding a comment reminding that the permissions to migrate became more strict in 4.19.

Reply:
Thanks @yossisegev. I am going to keep this as is.